package.json

```diff
@@ -37,7 +37,7 @@
     "yaml-front-matter": "^4.1.1"
   },
   "scripts": {
-    "spin": "nodemon --watch content --watch static --watch templates --watch spin-up-hub --ext html,md,rhai,hbs,css,js --verbose --legacy-watch --signal SIGINT --exec 'npm run build-index && npm run build-hub-index && spin up --file spin.toml --quiet --env PREVIEW_MODE=$PREVIEW_MODE'",
+    "spin": "nodemon --watch content --watch static --watch templates --watch spin-up-hub --ext html,md,rhai,hbs,css,js --verbose --legacy-watch --signal SIGINT --exec 'npm run build-index && npm run build-hub-index && spin watch --file spin.toml --quiet --env PREVIEW_MODE=$PREVIEW_MODE'",
```

Comment:
You do not need both `spin watch` and `spin up`. They effectively serve the same purpose of restartting. We got to choose one or the other,

Reply:
Hi @karthik2804 
Not sure I understand. There is only one spin command in the package.json (`spin watch`) so as far as I can tell after this update the site will exclusively use `spin watch`.

Reply:
I think this might be a typo for not needing both `spin watch` and  `nodemon --watch`.  If nodemon is already watching, it shouldn't be telling Spin to watch, just to run.  Conversely, if both are watching, `spin watch` get as far as going "oh there's a change I better restart `spin up`" and then nodemon will swoop in and kills it.